charts/gardener-extension-shoot-rsyslog-relp-admission/charts/runtime/templates/deployment.yaml

```diff
@@ -47,6 +47,9 @@ spec:
         {{- if .Values.kubeconfig }}
         - --kubeconfig=/etc/gardener-extension-shoot-rsyslog-relp-admission/kubeconfig/kubeconfig
         {{- end }}
+        {{- if .Values.projectedKubeconfig }}
```

Comment:
Shouldn't this be in an `else if` block or do we expect to never have both `.Values.kubeconfig` and `.Values.projectedKubeconfig` defined?

Reply:
Yes, we expect that both it not set at the same time since it doesn't make much sense.